front/app/components/admin/ResourceList/SortableList.tsx

```diff
@@ -44,15 +41,13 @@ const SortableList = ({
   const [itemsWhileDragging, setItemsWhileDragging] = useState<Item[] | null>(
     null
   );
-  const [updating, setUpdating] = useState(false);
   const prevItems = usePrevious(items);
 
   useEffect(() => {
-    if (updating && itemOrderWasUpdated(prevItems, items)) {
```

Comment:
`updating` was always `false` and not set to `true` anywhere, so `itemsWhileDragging` did not get reset.